.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# Documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
Nice work @OlaShabalina ! 🙌🏽 

I think we should set a more frequent interval schedule initially to give us more opportunities to test, observe and tweak dependabot's behaviour. Maybe execute `daily` for now? Once we are happy with how dependabot is running, we can move to a `weekly` schedule. 

What do you think @OlaShabalina ?